core/store/orm/orm.go

```diff
@@ -899,15 +899,45 @@ func (orm *ORM) TxAttempts(offset, limit int) ([]models.TxAttempt, int, error) {
 	return attempts, count, err
 }
 
-func (orm *ORM) UnconfirmedTxAttempts() ([]models.TxAttempt, error) {
+func (orm *ORM) HighestPricedUnconfirmedTxAttempts() ([]models.TxAttempt, error) {
 	var items []models.TxAttempt
+
+	subquery := orm.DB.
+		Select("tx_attempts.*").
+		Table("tx_attempts").
+		Joins("INNER JOIN txes ON txes.id = tx_attempts.tx_id").
+		Where("txes.confirmed = ?", false).
+		Order("tx_attempts.gas_price DESC").
```

Comment:
`gas_price character varying(78) NOT NULL`

gas_price is varchar, thus sorting by this would be lexical and incorrect, I think you'll want to migrate to a numeric with enough room for a 256 bit integer here. And a test please.